frontend/src/components/maps/ParcelPopupView.tsx

```diff
@@ -12,29 +12,47 @@ export interface IParcelDetailProps {
 export const ParcelPopupView = (props: IParcelDetailProps | null) => {
   const parcelDetail: IParcelDetail | null | undefined = props?.parcelDetail;
 
-  return <Container className="parcelPopup" fluid={true}>
-    {!parcelDetail ?
-      <Alert variant="danger">Failed to load parcel details.</Alert>
-      :
-      <Row>
-        <Col>
-          <ListGroup>
-            <ListGroup.Item><Label>Assessed Value: </Label>${parcelDetail?.assessedValue.toLocaleString('en-US', { minimumFractionDigits: 2 })}</ListGroup.Item>
-          </ListGroup>
-          <ListGroup>
-            <ListGroup.Item>
-              <p><Label>Address: </Label>{parcelDetail?.address?.line1}</p>
-              <p>{parcelDetail?.address?.city}, {parcelDetail?.address?.province} {parcelDetail?.address?.postal}</p>
-            </ListGroup.Item>
-            <ListGroup.Item><Label>PID: </Label> {parcelDetail?.pid}</ListGroup.Item>
-          </ListGroup>
-          <ListGroup>
-            <ListGroup.Item><Label>Agency: </Label>{parcelDetail?.agency}</ListGroup.Item>
-            <ListGroup.Item><Label>Classification: </Label>{parcelDetail?.classification}</ListGroup.Item>
-          </ListGroup>
-        </Col>
-      </Row>
-    }
-  </Container >
-
-}
+  return (
+    <Container className="parcelPopup" fluid={true}>
+      {!parcelDetail ? (
+        <Alert variant="danger">Failed to load parcel details.</Alert>
+      ) : (
+        <Row>
+          <Col>
+            <ListGroup>
+              <ListGroup.Item>
+                <Label>Assessed Value: </Label>$
+                {parcelDetail?.assessedValue.toLocaleString('en-US', { minimumFractionDigits: 2 })}
+              </ListGroup.Item>
+            </ListGroup>
+            <ListGroup>
+              <ListGroup.Item>
+                <p>
+                  <Label>Address: </Label>
+                  {parcelDetail?.address?.line1}
+                </p>
+                <p>
+                  {parcelDetail?.address?.city}, {parcelDetail?.address?.province}{' '}
+                  {parcelDetail?.address?.postal}
+                </p>
+              </ListGroup.Item>
+              <ListGroup.Item>
+                <Label>PID: </Label> {parcelDetail?.pid}
+              </ListGroup.Item>
+            </ListGroup>
+            <ListGroup>
+              <ListGroup.Item>
+                <Label>Agency: </Label>
+                {parcelDetail?.agency}
+              </ListGroup.Item>
+              <ListGroup.Item>
+                <Label>Classification: </Label>
+                {parcelDetail?.classification}
+              </ListGroup.Item>
+            </ListGroup>
+          </Col>
+        </Row>
+      )}
+    </Container>
+  );
+};
```

Comment:
The above is also based on rules about using parentheses to wrap components being returned in a function. Again, we are using default values of linting here. We can change/tweak specific rules if we don't like them